package.json

```diff
@@ -5,13 +5,29 @@
   "repository": "https://github.com/AliMD/One",
   "author": "Ali Mihandoost <alimihandoost@gmail.com>",
```

Comment:
please change author from `Ali Mihandoost <alimihandoost@gmail.com>` to `@one community`

Reply:
I do not dare 🤣

Reply:
😂😂🤣
but @asmanp2012 say do it (in meet) !